### ISSUE TITLE
Training Stuck at Step 3/336 with Wan 2.1 I2V 14B Training

### ISSUE THREAD

**Issue by hl2dm** [kohya-ss/musubi-tuner#163]
Hello,
I'm trying to train a Wan 2.1 I2V 14B model using musubi-tuner, but I'm experiencing an issue where the training process gets stuck at step 3/336 and doesn't progress any further.
My setup:

Windows environment

dataset.toml

[general]
resolution = [832, 480]
batch_size = 1
enable_bucket = true
bucket_no_upscale = false
caption_extension = ".txt"
 

[[datasets]]
video_directory = "D:/musbi/musubi-tuner/data/videos"
cache_directory = "D:/musbi/musubi-tuner/data/cache/videos"
target_frames = [1, 5, 9, 13, 17, 21, 25, 29, 33, 37, 41, 45, 49, 53, 57, 61, 65, 69, 73, 77, 81]
frame_extraction = "head"

step1

 python wan_cache_latents.py --dataset_config D:/musbi/musubi-tuner/dataset/dataset.toml --vae D:/musbi/musubi-tuner/wan/wan_2.1_vae.safetensors --clip D:/musbi/musubi-tuner/wan/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth

INFO:root:weights loaded from D:/musbi/musubi-tuner/wan/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth: <All keys matched successfully>
INFO:cache_latents:Encoding dataset [0]
21it [01:18,  3.76s/it]

step2

python wan_cache_text_encoder_outputs.py --dataset_config D:/musbi/musubi-tuner/dataset/dataset.toml --t5 D:/musbi/musubi-tuner/wan/models_t5_umt5-xxl-enc-bf16.pth --batch_size 16  

step3

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 wan_train_network.py --task i2v-14B --dit D:/musbi/musubi-tuner/wan/wanAIWan21VideoModelSafetensors_kijaiWan21I2V14B480P.safetensors --dataset_config D:/musbi/musubi-tuner/dataset/dataset.toml --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 2e-4 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 3.0 --max_train_epochs 16 --save_every_n_epochs 1 --seed 42 --output_dir D:/musbi/musubi-tuner/out --blocks_to_swap 36 --output_name ones4

stay on:

INFO:hv_train_network:set DiT model name for metadata: D:/musbi/musubi-tuner/wan/wan_2.1_vae2.safetensors
steps:   0%|                                                                                   | 0/336 [00:00<?, ?it/s]INFO:hv_train_network:DiT dtype: torch.float8_e4m3fn, device: cuda:0

epoch 1/16
Trying to import sageattention
Failed to import sageattention
Trying to import sageattention
Failed to import sageattention
INFO:dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1
INFO:dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1

if i chage resolution  to [512,512]

they well be Freeze on 3/336

Is this a known issue? Are there any common configuration problems that might cause training to stall at early steps? Do you have any recommendations for settings or parameters that might help resolve this?
I'd greatly appreciate any guidance you can provide.
Thank you!

**Comment by hl2dm** [comment]
Thank you for your reminder. After adjusting the target_frames parameter, the issue of it completely freezing has been resolved. I am still exploring the optimal configuration. Currently, I am using target_frames = [1, 5, 9, 13, 17, 21, 25, 29, 33] to train thirteen videos, each no more than 20 seconds long. Thank you for taking the time to answer my question.
